.github/workflows/compile_test.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
       matrix:
         flux: [ 0.52.0, 0.49.0]
-        mode: ["FLUX_RPC", "UCX"]
+        mode: ["UCX"]
```

Comment:
Create an issue regarding FLUX_RPC failure. Do you see it fail even when you run manually? It worked for me recently.

Reply:
Yes I got the same error as the CI, Maybe Me and the CI are missing something.